evm/src/main/java/org/hyperledger/besu/evm/precompile/AbstractBLS12PrecompiledContract.java

```diff
@@ -67,6 +67,20 @@ public abstract class AbstractBLS12PrecompiledContract implements PrecompiledCon
     this.inputLimit = inputLen + 1;
   }
 
+  /**
+   * Is bls12 supported on this platform
+   *
+   * @return true if the native library was loaded.
+   */
+  public static boolean isAvailable() {
+    try {
+      return LibGnarkEIP2537.ENABLED;
+    } catch (UnsatisfiedLinkError | NoClassDefFoundError ule) {
+      LOG.info("bls12-381 native precompile not available: {}", ule.getMessage());
```

Comment:
Think it would be useful to also add the log to BesuCommand.configureNativeLibs that we _are_ using the native version (including which library: Gnark, Constantine, etc??)

Reply:
Yeah, definitely.  I will hold off on that til the next pr that does the auto-selecting based on arch since it will change in that one significantly